src/pages/logIn/LogInPage.tsx

```diff
@@ -81,20 +118,19 @@ export default function LogInPage() {
               id="password"
               className={cx("col-span-2")}
               type="password"
+              validation={passwordValidation}
               maxLength={200}
-              required
-              value={password}
-              onChange={handleChangePassword}
             />
             <div />
             <Button type="submit">Log in</Button>
           </Box>
-        </form>
+        </Form>
       )}
       {status === "new" && isLoggedIn && <p>You are already logged in.</p>}
       {status === "error/user-not-found" && (
         <p>
-          Sorry, there is no Nuffshell user with email address <em>{email}</em>.
+          Sorry, there is no Nuffshell user with email address{" "}
+          <em>{formState.email}</em>.
```

Comment:
For security reasons, I would avoid to give useful information messages to possible attackers like:

"Sorry, there is no Nuffshell user with email address"

and I would go for a generic error messages in for cases like: 

1. User already exist
2. there is no Nuffshell user with email address

or similar.

If you remove the formState.email message you can get rid of 

  const [formState, setFormState] = useState(initialFormState);

  onReset={updateFormState}
  onChange={updateFormState}
  onInit={updateFormState}

so it becomes: 

```js
<Form onSubmit={handleSubmit} initialState={initialFormState}>
```

Reply:
Agreed, done, thanks for the tip!